clients/tests/CMakeLists.txt

```diff
@@ -96,7 +96,7 @@ foreach(i RANGE 0 ${len1})
   if(NOT EXISTS "${CMAKE_MATRICES_DIR}/${mat}.bin")
     message("-- Downloading and extracting test matrix ${m}.tar.gz")
     file(DOWNLOAD https://sparse.tamu.edu/MM/${m}.tar.gz ${CMAKE_MATRICES_DIR}/${mat}.tar.gz
-        INACTIVITY_TIMEOUT 3
+        INACTIVITY_TIMEOUT 10
```

Comment:
Consistently fails downlaoding matrices because of timeouts on windows. Increasing to 10 seconds dramatically reduces failures.